lsp/src/main/java/com/salesforce/slds/lsp/codeactions/CodeActionConverter.java

```diff
@@ -59,8 +59,9 @@
                         return createAlternativeTokenCodeAction(params, info.get());
                     case UTILITY_TOKENS:
                         return createUtilityTokenCodeAction(params, info.get());
-                    case NON_MOBILE_FRIENDLY:
-                        return createNonMobileFriendlyBaseComponentCodeAction(params, info.get());
+                    case MOBILE_SLDS:
```

Comment:
Just confirm, we will update this before updating the client. Is that correct?

Reply:
@billyma In my PR I am removing this comment and adding code actions for suppressing these warnings via the Quick Fix menu (if that's what you're asking about).

Reply:
Yes, just want to confirm your change will be in before we update the client. So, the experience won't be weird for the user.